src/components/inputs/Input.tsx

```diff
@@ -18,7 +18,8 @@ const useStyles = makeStyles({
     }
 });
 
-export const Input = () => {
+export const Input = (props:any) => {
+    const {label, type} = props;
```

Comment:
you can use restructuring in a function 
export const Input = ({label, type}) =>

Reply:
if you don't always want to specify the type, write 
export const Input = ({label, type="text"}) =>

Reply:
then if you dont pass type, "text" will be as default

Reply:
fixed